projects/adf-office-services-ext/src/lib/evaluators.spec.ts

```diff
@@ -29,6 +29,7 @@ describe('evaluators', () => {
   describe('canOpenWithOffice', () => {
     it('should return [false] if using SSO', () => {
       const context: any = {
+        aosPlugin: true,
```

Comment:
it should not be necessary to add the true value, the plugin should be enabled is the plugin. aosPlugin is true or the property is missing to be backward compatible